staging/src/k8s.io/apiserver/pkg/apis/audit/validation/validation.go

```diff
@@ -17,6 +17,9 @@ limitations under the License.
 package validation
 
 import (
+	"strings"
+
+	genericvalidation "k8s.io/apimachinery/pkg/api/validation"
```

Comment:
This package is also named validation, rename the imported one to distinguish them.
Do we have some convention for this?